examples/ruby/gift_giver/app/controllers/refresh_controller.rb

```diff
@@ -1,18 +1,24 @@
 class RefreshController < ApplicationController 
-    URL = 'https://api.meetup.com/valetechtalks/events/268854460/rsvps?&sign=true&photo-host=public&fields=answers'
+  URL = 'https://api.meetup.com/valetechtalks/events/268854460/rsvps?&sign=true&photo-host=public&fields=answers'
 
-    def index
-        json = JSON.parse(::HTTParty.get(URL).body)
-        names = json.map { |item| item['member']['name'] }
-        names.each do |name|
-            Attendee.create!(
-              name: name,
-              awarded: false,
-              email: "#{name}@example.com"
-            )
+  def index
```

Comment:
I would move all the logic inside here to another class. Since we are focused on the tests we need to avoid coupling here and move this to another class which will be easier for testing.

Reply:
I'm considering let this to refactor as part of writing tests,
what do you think?

Reply:
Agreed.